.gitlab-ci.yml

```diff
@@ -1,4 +1,4 @@
-image: python:3.6
+image: xbrianh/dss-build-box  # TODO: Move to humancellatlas docker account?
```

Comment:
1. Can we move this to the humancellatlas account?

2. Can you put in a comment a link to instructions for how to create and upload this image?

Reply:
1. Maybe
2. will do